### ISSUE TITLE
Song title is truncated

### ISSUE THREAD

**Issue by enwokoma** [Charlieras262/BeatPlayer#13]
Some of the Song titles are truncated in the **SongDetailFragment** 
![Screenshot_20200529-155830](https://user-images.githubusercontent.com/26168869/83274426-b5444180-a1c5-11ea-9a79-72efc14ed4ff.png)

Adding the marquee attribute as seen below should resolve this and make the song title to scroll automatically when the text is too long.
```
songTitle.ellipsize = TextUtils.TruncateAt.MARQUEE
songTitle.isSingleLine = true
songTitle.marqueeRepeatLimit = -1
songTitle.isSelected = true
```

This is how it looks after adding marquee to it
![Screenshot_20200529-160049](https://user-images.githubusercontent.com/26168869/83274429-b6756e80-a1c5-11ea-82b9-cddd82103460.png)

But, even with the marquee, the song title only scrolls when the song is paused but does not scrolling when the song is playing

**Comment by Charlieras262** [comment]
Hi @enwokoma, i'm working on this issue but i'm having some problems because when i add marquee to the title it stucks, but i already identified the problem but i'm working on the solution then a will fix this issue.

**Comment by Charlieras262** [comment]
Hi, I already pushed i commit to solved this issue (#13).
